src/world-company-remuneration/salary-calculator-service-impl.ts

```diff
@@ -3,6 +3,20 @@ const K = "K";
  * @author Jack Napier <jnapier@therealjoker.com>
  */
 export function salary(a: number, b: number) {
+    if (a === null){
+        if (666 === a) {
+            return "132" + K;
+        } else {
+            if (b > 10) {
+                return "41,8" + K;
```

Comment:
In my opinion we should be explicit here and extract out the 10% in a constant 
`const  newComersAdvandage  = 0,1`
and take the salary grid from the corresponding seniority, ex: `JUNIOR.salary`
and apply it :
```
const newComerSalary = JUNIOR.salary * (1+newComersAdvantage)
return ${newComerSalary}K` (no need to extract out K)
```